src/gui/components/wxGameList.cpp

```diff
@@ -142,6 +143,60 @@ void wxGameList::LoadConfig()
 	}
 }
 
+// for unknow reasons of wxWidgets, there are some issues if we change column size in a SizeEvent handler func when use sizer-based layout.
+// the list may not redraw correctly, or extra blank rows/columns shown in list...
+// So we DO NOT change column size here, just send a ColumnResize event to ColumnResize handler, resize will be done over there.
```

Comment:
I'd prefer these comments to be inside the function. Also, it could be shortened a good bit.

I do appreciate that it explains your workaround/fix for me as a reviewer.